explorer/ui/app/components/home/blockDataTable.tsx

```diff
@@ -5,16 +5,17 @@ import { XBlock } from '~/graphql/graphql'
 import { ColumnDef } from '@tanstack/react-table'
 import SimpleTable from '../shared/simpleTable'
 import { GetBlockCount, GetBlocksInRange } from '../queries/block'
+import { dateFormatter } from '~/lib/formatting'
 
 export const loader = async () => {
-  return json<XBlock[]>(new Array())
+  return json<XBlock[]>([])
 }
 
 export default function XBlockDataTable() {
-  const d = useLoaderData<typeof loader>()
+  useLoaderData<typeof loader>()
 
-  let rows = GetBlocksInRange(0, 1000)
-  let count = GetBlockCount()
+  const rows = GetBlocksInRange(0, 1000)
+  const count = GetBlockCount()
```

Comment:
same comment on `use` naming here - also,seems we have setup to `useLoaderData` - let's move queries there